### PR TITLE
Keynote blog post

### PR DESCRIPTION
**Pull request by tedmcox**
Added blog post announcing keynote speech.

### CODE REVIEW

**Comment by ex-nerd** [comment]
Should probably do a single post to cover both keynotes.  Do we have any info about Shauna?

**Comment by tedmcox** [comment]
D'oh. Somehow I missed that we had two keynotes this year. If you send me a
link to Shauna's info I'll edit the post and resubmit.

**Comment by tedmcox** [comment]
I edited the post to show info for both Shauna and Richard. Renamed the file to something more generic.

File: _posts/2015-06-14-Keynotes.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Besides her work bringing open source tools to some of the world's top 
+universities, Shauna's numerous contributions to the open source community are 
+listed on the [Open Hatch](http://campus.openhatch.org/about/) site:
+
+>She's also a freelance programmer, researcher, organizer and writer. She's
+>worked for the MIT Media Lab and Civic Commons, among others, and volunteers
+>with the Sunlight Foundation and the Open Science Collaboration.
+
+Richard's three decades of activism is summed up on the [Free Software
```
Comment (ex-nerd) [inline _posts/2015-06-14-Keynotes.md]: s/is/are/

File: _posts/2015-06-14-Keynotes.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+We are excited to announce that Shauna Gordon-McKeon, the main organizer of 
+OpenHatch's [Open Source Comes to Campus events](http://campus.openhatch.org/), 
+and Richard M. Stallman, GNU's Project leader, will deliver the keynote 
+addresses at the 2015 Seattle GNU/Linux Conference.
+
+Besides her work bringing open source tools to some of the world's top 
+universities, Shauna's numerous contributions to the open source community are 
+listed on the [Open Hatch](http://campus.openhatch.org/about/) site:
```
Comment (ex-nerd) [inline _posts/2015-06-14-Keynotes.md]: s/:$/./

**Comment by ex-nerd** [comment]
Just one little grammar check, then it looks great to me.

**Comment by tedmcox** [comment]
Fixed.

**Pull request merged**